scripts/ci/jobs/check-log-query-data.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```

Comment:
This won't run right now. There's a PR that still needs to made in openshift/release